src/tools/processData.js

```diff
@@ -0,0 +1,301 @@
+        context: await processHtml(report.context),
+      }),
+    );
+    if (imported) {
+      const newImported = {
+        additionalNotesForThisActivity: await processHtml(
+          imported.additionalNotesForThisActivity,
+        ),
+        cdiGranteeName: await processHtml(imported.cdiGranteeName),
+        contextForThisActivity: await processHtml(
+          imported.contextForThisActivity,
+        ),
+        created: imported.created,
+        createdBy: convertEmails(imported.createdBy),
+        duration: imported.duration,
+        endDate: imported.endDate,
+        format: imported.format,
+        goal1: imported.goal1,
+        goal2: imported.goal2,
+        granteeFollowUpTasksObjectives: await processHtml(
+          imported.granteeFollowUpTasksObjectives,
+        ),
+        granteeName: convertGranteeName(imported.granteeName),
+        granteeParticipants: imported.granteeParticipants,
+        granteesLearningLevelGoal1: imported.granteesLearningLevelGoal1,
+        granteesLearningLevelGoal2: imported.granteesLearningLevelGoal2,
+        manager: convertEmails(imported.manager),
+        modified: imported.modified,
+        modifiedBy: convertEmails(imported.modifiedBy),
+        multiGranteeActivities: imported.multiGranteeActivities,
+        nonGranteeActivity: imported.nonGranteeActivity,
+        nonGranteeParticipants: imported.nonGranteeParticipants,
+        nonOhsResources: imported.nonOhsResources,
+        numerOfParticipants: imported.numberOfParticipants,
+        objective11: imported.objective11,
+        objective11Status: imported.objective11Status,
+        objective12: imported.objective12,
+        objective12Status: imported.objective12Status,
+        objective21: imported.objective21,
+        objective21Status: imported.objective21Status,
+        objective22: imported.objective22,
+        objective22Status: imported.objective22Status,
+        otherSpecialists: convertEmails(imported.otherSpecialists),
+        otherTopics: imported.otherTopics,
+        programType: imported.programType,
+        reasons: imported.reasons,
+        reportId: imported.reportId,
+        resourcesUsed: imported.resourcesUsed,
+        sourceOfRequest: imported.sourceOfRequest,
+        specialistFollowUpTasksObjectives: await processHtml(
+          imported.specialistFollowUpTasksObjectives,
+        ),
+        startDate: imported.startDate,
+        tTa: imported.tTa,
+        targetPopulations: imported.targetPopulations,
+        topics: imported.topics,
+        ttaProvidedAndGranteeProgressMade: imported.ttaProvidedAndGranteeProgressMade,
+      };
+      promises.push(report.update({ imported: newImported }));
+    }
+  }
+
+  for await (const file of files) {
+    promises.push(
+      file.update({
+        originalFileName: convertFileName(file.originalFileName),
+      }),
+    );
+  }
+
+  for await (const hsesUser of hsesUsers) {
```

Comment:
Should we give these users "admin" permissions?

Reply:
Yes, I added the "admin" permissions along with read/write and approve reports for region 1. Good catch.